src/encoded/schemas/changelogs/library.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for library.json
 
+### Schema version 20
+* Added *construction_method* enum *capped RNA enrichment*, *uncapped RNA enrichment*, and *PCR-free*.
```

Comment:
I think you need to say that the property changed to be an array

Reply:
Thanks for pointing this out, I added "The property is now also submittable as an array."